lua/triptych/mappings.lua

```diff
@@ -1,3 +1,4 @@
+local tbl = require 'plenary.tbl'
```

Comment:
I'm actually trying to reduce my dependency on plenary. Instead of using this, can we either change the `opts` param to a description string, or use `utils.merge_tables`?

Reply:
I switched to `utils.merge_tables` and swapped the arguments.